lib/ts/recipe/oauth2/recipeImplementation.ts

```diff
@@ -16,10 +16,14 @@
 import NormalisedURLPath from "../../normalisedURLPath";
 import { Querier } from "../../querier";
 import { NormalisedAppinfo } from "../../types";
-import { toCamelCase } from "../../utils";
+import { toSnakeCase, transformObjectKeys } from "../../utils";
 import { OAuth2Client } from "./OAuth2Client";
 import { RecipeInterface, TypeNormalisedInput } from "./types";
 
+function createOAuth2ClientFromResponse(response: any): OAuth2Client {
+    return new OAuth2Client(transformObjectKeys(response, "camelCase"));
+}
```

Comment:
I'd prefer if this was defined on `OAuth2Client`

Reply:
Done